libs/ha/ha_common_libs_k8s.py

```diff
@@ -1471,15 +1468,17 @@ def get_node_resource_ids(node_obj, r_type, n_type=None, node_id=None):
     def delete_kpod_with_shutdown_methods(self, master_node_obj, health_obj,
                                           pod_prefix=None, kvalue=1,
                                           down_method=common_const.RESTORE_SCALE_REPLICAS,
-                                          event=None):
+                                          event=None, event_set_clr=None):
```

Comment:
Here also, please mention default value type list.

Reply:
This give warning of Default argument value is mutable